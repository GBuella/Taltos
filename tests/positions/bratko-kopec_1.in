force
nopost
setboard 1k1r4/pp1b1R2/3q2pp/4p3/2B5/4Q3/PPP2B2/2K5 b - -
nps 100000
st 2
search_sync
hash_value_min 20000
