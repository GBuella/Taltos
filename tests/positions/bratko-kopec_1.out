1. ... Qd1+
ok
